modules/ROOT/pages/endpoint-as2-receive.adoc

```diff
@@ -1,75 +1,81 @@
 = AS2 Receive Endpoint Settings
 
-AS2 endpoint settings specify where to receive inbound files from your trading partners. You can use the same AS2 receive endpoints to receive inbound transmissions from multiple partners.
+AS2 endpoint settings dictate where to receive inbound files from your trading partners. You can use the same AS2 receive endpoints to receive inbound transmissions from multiple partners.
```

Comment:
"AS2 receive endpoint settings dictate...", per "You can use the same AS2 receive endpoints to..."?
Are we specifically making the settings distinct from the endpoints themselves?
Also, would this be simpler and just as accurate:
```suggestion
Your AS2 endpoint settings determine where you receive inbound data from your trading partners. One AS2 endpoint can receive inbound data from multiple partners.
```
Is "you" = "host"? Clarify both above and below, and throughout these topics.